hkmc2/shared/src/test/mlscript/ucs/examples/BinarySearchTree.mls

```diff
@@ -34,7 +34,7 @@ fun insert(t, v) = if t is
   Node(v', l, r) and
     v < v' then Node(v', insert(l, v), r)
     v > v' then Node(v', l, insert(r, v))
-    _ then t
```

Comment:
Does this no longer work? Any reason for changing it?

Reply:
Reverted in 55da3fe188a9d236f6f5816102382dc7300c8dd3.